project/DistributionPackage.scala

```diff
@@ -278,6 +278,36 @@ object DistributionPackage {
     exitCode == 0
   }
 
+  def runProjectManagerPackage(
+    engineRoot: File,
+    distributionRoot: File,
+    args: Seq[String],
+    log: Logger
+  ): Boolean = {
+    import scala.collection.JavaConverters._
+
+    val enso = distributionRoot / "bin" / "project-manager"
```

Comment:
To continue the [idea of the previous comment](https://github.com/enso-org/enso/pull/7181/files#r1248477158) here we would need to invoke JVM with `project-manager.jar`. Necessary for debugging the project manager, but also useful for _plain execution_ as building _native image_ takes a lot of time.